### ISSUE TITLE
PHP include inline editor

### ISSUE THREAD

**Issue by lxsymington** [adobe/brackets#11809]
Just thought given the existing css inline editor functionality this may be a <em>relatively</em> simple enhancement. Also given that php is used only a negligible amount less than css and js a fair number of other users may find this functionality useful. Having looked though the contribution pages I feel this is a bit beyond me. Anyway essentially where one may have something like this:
```php
<footer>
    <?php include('includes/footer.php'); ?>
</footer>
```
making the inline editor available so upon calling the quick edit bring the content of the include in, much like with the css inline editor:

```php
<footer>
    <?php include('includes/footer.php'); ?>
```
<hr>
```html
    <a href="info.php" title="more info">Click Me</a>
    <script src="app.js" type="text/javascript"></script>
    <script src="lib.js" type="text/javascript"></script>
```
<hr>
```php
</footer>
```

this would also mean that when editing the include one could still use the live preview effectively, since the user would not have navigated away from the main page. Whereas currently when editing a php include the live preview obviously doesn't have any styles referenced, slightly diminishing it's usefulness in that situation.

**Comment by courcoa** [comment]
I'm 100% agree with you.
It's the only things I'm waiting to do Brackets my favorite and only text editor.

**Comment by lxsymington** [comment]
If this could work with php functions as well, it would probably be useful to even more users, but I imagine getting fairly complex and this all probably relies on third party localhosts?

**Comment by Necrospasm** [comment]
Yes please! When you have a large amount of files open each having a couple thousand lines of code and hundreds of functions, finding a function is a pain in the ass. Been looking for an extension everywhere.
